radiation/FatesRadiationDriveMod.F90

```diff
@@ -438,11 +439,18 @@ subroutine FatesSunShadeFracs(nsites, sites,bc_in,bc_out)
                            if_notair: if (ft>0) then
                               area_frac = twostr%scelg(cl,icol)%area
                               vai = twostr%scelg(cl,icol)%sai+twostr%scelg(cl,icol)%lai
-                              nv = minloc(dlower_vai, DIM=1, MASK=(dlower_vai>vai))
+
+                              nv = GetNVegLayers(vai)
+
                               do iv = 1, nv
                                  
-                                 vai_top = dlower_vai(iv)-dinc_vai(iv)
-                                 vai_bot = min(dlower_vai(iv),twostr%scelg(cl,icol)%sai+twostr%scelg(cl,icol)%lai)
+                                 vai_top = dlower_vai(iv)
```

Comment:
Is it correct to say that `vai_top` has shifted now that the lower bound value is zero?

Reply:
vai_top still has the same meaning, but dlower_vai has changed.  Lower used to refer to lower vertical position, now it refers to the numericaly lower edge of the bin.  So vai_top points directly to dlower_vai now since they conceptually align.